tooling-tests/src/test/resources/test-src/input/tool_test_generate_15/target/persist_db_scripts.sql

```diff
@@ -0,0 +1,26 @@
+DROP TABLE IF EXISTS MultipleAssociations;
```

Comment:
Same

Reply:
updated in [eac1fc8](https://github.com/ballerina-platform/persist-tools/pull/23/commits/eac1fc89ac6d1d3cd9a9387fce669b33074b9ced)